plugins/outputs/nats/nats_test.go

```diff
@@ -19,6 +21,7 @@ func TestConnectAndWriteIntegration(t *testing.T) {
 	servicePort := "4222"
 	container := testutil.Container{
 		Image:        "nats",
+		Cmd:          []string{"--js"},
```

Comment:
Please have two integration tests, one with the current behavior and one with jetstream enabled.

Reply:
Combined them into one using table driven test suite.